### PR TITLE
Add JupyterLab docs

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

File: docs/hub/spaces-sdks-docker-jupyter.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* The **Hardware**: the hardware you want to use for your JupyterLab instance. This goes from CPUs to H100s.
+
+* You can optionally configure a `JUPYTER_TOKEN` password to protect your JupyterLab workspace. When unspecified, defaults to `huggingface`. We strongly recommend setting this up if your Space is public or if the Space is in an organization.
+
+<Tip warning={true}}>
+
+Storage in Hugging Face Spaces is ephemeral, and the data you store in the default configuration can be lost in a reboot or reset of the Space. We recommend to save your work to a remote location or to use persistent storage for your data.
+
+</Tip>
+
```
Comment (apolinario) [inline docs/hub/spaces-sdks-docker-jupyter.md:34]:
````suggestion
### Setting up persistent storage

To set up persitent storage on the Space, you go to the Settings page of your Space and choose one of the options: `small`, `medium` and `large`. Once persisent storage is set up, it gets mounted in `/data`. In order to see the persistent folder in the JupyterLab UI, you can create a `symlink` for it in the JupyterLab Terminal.

```shell
ln -s /data persistent_folder
```

````

File: docs/hub/spaces-sdks-docker-jupyter.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Read more
+
+- [HF Docker Spaces](https://huggingface.co/docs/hub/spaces-sdks-docker)
+
+If you have any feedback or change requests, please don't hesitate to reach out to the owners on the [Feedback Discussion](https://huggingface.co/spaces/SpacesExamples/jupyterlab/discussions/3).
```
Comment (apolinario) [inline docs/hub/spaces-sdks-docker-jupyter.md:40]:
```suggestion
If you have any feedback or change requests, please don't hesitate to reach out to the owners on the [Feedback Discussion](https://huggingface.co/spaces/SpacesExamples/jupyterlab/discussions/3).

## Acknowledgments

This template was created by [camenduru](https://twitter.com/camenduru) and [nateraw](https://huggingface.co/nateraw), with contributions of [osanseviero](https://huggingface.co/osanseviero) and [azzr](https://huggingface.co/azzr).
```

File: docs/hub/spaces-sdks-docker-jupyter.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Acknowledgments
+
+This template was created by [camenduru](https://twitter.com/camenduru) and [nateraw](https://huggingface.co/nateraw), with contributions of [osanseviero](https://huggingface.co/osanseviero) and [azzr](https://huggingface.co/azzr).
```
Comment (pcuenca) [inline docs/hub/spaces-sdks-docker-jupyter.md:44]:
```suggestion
This template was created by [camenduru](https://twitter.com/camenduru) and [nateraw](https://huggingface.co/nateraw), with contributions from [osanseviero](https://huggingface.co/osanseviero) and [azzr](https://huggingface.co/azzr).
```

maybe?

File: docs/hub/spaces-sdks-docker-jupyter.md (hunk 1 above)
Comment (osanseviero) [reply docs/hub/spaces-sdks-docker-jupyter.md:34]: Would you like to change the description given the last changes?

**Review by apolinario — APPROVED** [review]

File: docs/hub/spaces-sdks-docker-jupyter.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+* The **Hardware**: the hardware you want to use for your JupyterLab instance. This goes from CPUs to H100s.
+
+* You can optionally configure a `JUPYTER_TOKEN` password to protect your JupyterLab workspace. When unspecified, defaults to `huggingface`. We strongly recommend setting this up if your Space is public or if the Space is in an organization.
+
+<Tip warning={true}}>
+
+Storage in Hugging Face Spaces is ephemeral, and the data you store in the default configuration can be lost in a reboot or reset of the Space. We recommend to save your work to a remote location or to use persistent storage for your data.
+
+</Tip>
```
Comment (pcuenca) [inline docs/hub/spaces-sdks-docker-jupyter.md:33]:
```suggestion
</Tip>

```

**Review by pcuenca — APPROVED** [review]

**Pull request merged**